virtual_rainforest/models/abiotic/abiotic_tools.py

```diff
@@ -0,0 +1,70 @@
+"""The ``models.abiotic.models`` module contains a set of general functions and
```

Comment:
"models.abiotic.abiotic_tools"?

Reply:
Thanks, I noticed a few of them and made a small update